pandas/core/window/rolling.py

```diff
@@ -487,13 +504,33 @@ def _get_window_indexer(self, window: int) -> BaseIndexer:
             return VariableWindowIndexer(index_array=self._on.asi8, window_size=window)
         return FixedWindowIndexer(window_size=window)
 
+    def _apply_series(self, homogeneous_func: Callable[..., ArrayLike]) -> "Series":
+        """
+        Series version of _apply_blockwise
+        """
+        from pandas import Series  # TODO: use _constructor?
```

Comment:
Can you use what is commented here? We've had a few other PRs fixing that recently in similar code locations like groupby so good to keep consistent

Reply:
done